charts/comet-api-v1/templates/role.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "comet-api.roleName" . }}
```

Comment:
just inline the name here, no need to use complicated _helpers.tpl

Reply:
3cb1fef645ee21c6d753b0ab95a6a93b88eefce3